Mathlib/RingTheory/PowerSeries/NoZeroDivisors.lean

```diff
@@ -0,0 +1,77 @@
+/-
+Copyright (c) 2025 Anatole Dedecker. All rights reserved.
```

Comment:
My understanding is that the copyright for a file containing content from another file should be identical to the file it was taken from, but I'm not certain.